pkgs/tools/misc/scriptcommunicator/default.nix

```diff
@@ -0,0 +1,87 @@
+{ stdenv
+, fetchurl, qmake, pkgconfig
+, libpng, zlib, unzip
+, qtbase, qtscript, qttools, qtserialport
+, makeWrapper, makeDesktopItem
+}:
+
+let
+  pname = "scriptcommunicator";
+  version = "05.08";
+
+  desktopItem = makeDesktopItem {
+    name = "scriptcommunicator";
+    exec = "${pname}";
+    icon = "$out/share/scriptcommunicator/images/main_32x32.png";
+    comment = "Scriptable data terminal"; 
+    desktopName = "scriptcommunicator";
+    genericName = "scriptcommunicator";
+    categories = "Network;";
+  };
+
+in stdenv.mkDerivation rec {
+
+  name = "${pname}-${version}";
+
+  nativeBuildInputs = [ unzip qmake ];
+  buildInputs = [ qtbase qtserialport qtscript makeWrapper ];
+
+  # we want QtDesigner available to create UI
+  propagatedBuildInputs = [ qttools ];
+
+  src = fetchurl {
+      url = "mirror://sourceforge/projects/scriptcommunicator/files/Source/ScriptCommunicator_05_08_source.zip";
+      sha256 = "f5ae8b1d1031c0d944fdbdd25d23288970e2e1818d214e8eec1b1f49c20f0f4d";
```

Comment:
Hash doesn't work anymore. Please update in conjunction with the version update.